client/test-worker/turbo-resolver/resolver.spec.js

```diff
@@ -24,3 +24,18 @@ test('Resolver resolves vue2 deps', async t => {
   t.deepEqual(Object.keys(result.appDependencies), ['vue']);
   t.equal(Object.keys(result.resDependencies).length, 0);
 });
+
+test('Resolver resolves invalid deps', async t => {
+  t.timeoutAfter(5000);
```

Comment:
If your private jsdelivr clone could not return in 500ms, that's sounds like an issue you need to fix.

Reply:
Haha, don't worry, I put this in case someone removed the `.catch(reject)` in the resolver ;-).